src/v2/Components/UserSettings/UserSettingsTabs.tsx

```diff
@@ -42,6 +42,22 @@ const tabClass = (tabUrl, route) => {
   return currentRoute === tabUrl ? "active" : undefined
 }
 
+const useScrollIntoView = (isRouteActive: boolean) => {
+  const ref = React.useRef() as React.MutableRefObject<HTMLDivElement>
+  React.useEffect(() => {
+    if (isRouteActive && ref?.current) {
+      ref.current?.scrollIntoView &&
+        ref.current.scrollIntoView({
+          behavior: "smooth",
+          block: "end",
+          inline: "nearest",
+        })
+    }
+  }, [isRouteActive])
+
+  return { ref }
+}
```

Comment:
Note that we already have [this helper](https://github.com/artsy/force/blob/3afca276d9dcc6050e762264fc8497511e5ab8da/src/v2/Utils/scrollHelpers.ts#L15), and we use by attaching an `#scrollTo--whatever` id directly on the target dom node. Short and sweet. 

That said, if we need to use something more dynamic with reffs and so on, feel free to add a new function within the `scrollHelpers` file linked above.

Reply:
I moved the hook into `scrollHelpers`. Tried to use [this helper](https://github.com/artsy/force/blob/3afca276d9dcc6050e762264fc8497511e5ab8da/src/v2/Utils/scrollHelpers.ts#L15) but in this case it wasn't working (I guess because the scroll area is nested in another one and it is horizontal).

But if I am missing something and it can be done with [this helper](https://github.com/artsy/force/blob/3afca276d9dcc6050e762264fc8497511e5ab8da/src/v2/Utils/scrollHelpers.ts#L15) I would love to reuse that.

Reply:
The hook seems useful! all good 👍